### PR TITLE
Update to master substrate, clean up of modules, restructure repo

### PR DESCRIPTION
**Pull request by drewstone**

### CODE REVIEW

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -83,16 +69,17 @@ use constants::{time::*, currency::*};
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
-/// This runtime version.
+/// Runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("edgeware"),
+	spec_name: create_runtime_str!("node"),
```
Comment (raykyri) [inline node/runtime/src/lib.rs]: I believe the spec name should be `edgeware`.

File: node/runtime/src/lib.rs [hunk 2]
```diff
@@ -240,44 +236,49 @@ impl session::historical::Trait for Runtime {
...
 	type SessionInterface = Self;
-        type RewardCurve = RewardCurve;
+	type RewardCurve = RewardCurve;
 }
 
 parameter_types! {
-	pub const LaunchPeriod: BlockNumber = 3 * 24 * 60 * MINUTES;
-	pub const VotingPeriod: BlockNumber = 3 * 24 * 60 * MINUTES;
-	pub const EmergencyVotingPeriod: BlockNumber = 1 * 24 * 60 * MINUTES;
+	pub const LaunchPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
```
Comment (raykyri) [inline node/runtime/src/lib.rs]: I think we should use a 3-day launch period, 3-day voting period, 24-hour enactment period, 24-hour emergency voting period, and 3-day cooloff period for testnet.

File: node/runtime/src/lib.rs (hunk 1 above)
Comment (drewstone) [reply node/runtime/src/lib.rs]: will change.

File: node/runtime/src/lib.rs (hunk 2 above)
Comment (drewstone) [reply node/runtime/src/lib.rs]: will change.

**Pull request merged**